technical-reference/xslt-transform-engine-grid-property.md

```diff
@@ -91,7 +91,7 @@ Global level XSLT transform engine will be used if "Undefined" is selected for m
 
 Only standard XSLT and XPath instructions are supported in Saxon if claimed. Saxon also provides several different ways of extensions, most of which are only available in PE and EE, create custom XSLT transform based on these editions if you are willing to use Saxon PE and EE.
 
-**Custom Extension XML** is still a supported way for creating your custom extension for Saxon 9 HE transform engine. Create custom .Net extension functions by implementing ExtensionFunction or ExtensionFunctionDefinition intefaces, and add your implementations into **Custom Extension XML**.  Saxon 9 HE transform engine will register extension functions defined in **Custom Extension XML**, and transform processor can then recognize and invoke any call from XSLT.
+**Custom Extension XML** is still a supported way for creating your custom extension for Saxon 9 HE transform engine. Create custom .Net extension functions by implementing interface ExtensionFunction or ExtensionFunctionDefinition, and add your implementations into **Custom Extension XML**.  Saxon 9 HE transform engine will register extension functions defined in **Custom Extension XML**, and transform processor can then recognize and invoke any call from XSLT.
```

Comment:
suggestion:
if 'ExtensionFunction' and 'ExtensionFunctionDefinition' are the actual names of the interfaces, then I think it will be good if they are in different formatting, perhaps the way code shows up

Reply:
I dont think i can make them the way code shows, but different formatting would be a good idea. Thanks.

Reply:
if you put the text in back ticks it will show up like code:
![image](https://user-images.githubusercontent.com/45184761/72111673-57838b80-32f0-11ea-9b0f-3f8fcbb8dc00.png)

Reply:
Got you, thanks.